pythonFiles/aggregateTestResults.py

```diff
@@ -11,8 +11,8 @@
 # %%
 def getRuns(createdDate):
     runsResponse = requests.get(
-        "https://api.github.com/repos/microsoft/vscode-jupyter/actions/runs",
-        params={"event": "push", "created": createdDate},
+        "https://api.github.com/repos/microsoft/vscode-jupyter/actions/workflows/build-test.yml/runs?per_page=50",
```

Comment:
Github api looks like there is a query parameter for branch. Can we use that?

Reply:
yes, I thought that would refer to the wrong branch attribute, but it seems to only select the ones we want